setup.sh

```diff
@@ -19,6 +19,9 @@ FL_HOCKEY_TEAMS="" #Hockey ID (number not name)
 FL_HOCKEY_NOTIFY= #Email team when new build avialable? 0 = No, 1 = Yes
 ICON_OVERLAY_ASSETS_BUNDLE="" #Path to .xcassets
 TAB_USE_TIME_FOR_BUILD_NUMBER= #Use Time and date for build number or BUILD_NUMBER environment variable (created by jenkis or team city) true = use time, false = use BUILD_NUMBER
+ITUNES_CONNECT_USERNAME="" #iTunes Connect login (usually email address)
+ITUNES_CONNECT_TEAM_ID="" #The ID of your iTunes Connect team if you're in multiple teams
```

Comment:
Can we add this link somewhere on how to find this value? https://github.com/fastlane/fastlane/issues/4301#issuecomment-253461017